sim/processed.py

```diff
@@ -0,0 +1,45 @@
+
+
+class ProcessedSample:
+    """ simulated genomes combined with and btb-seq sequence data 
+        Useful for making comparisons when post-processing results
+    """
+
+    def __init__(self, simulated_genome, sequenced_sample):
+        if simulated_genome.name != sequenced_sample.name:
+            raise Exception("Sample names do not match")
+
+        self.name = simulated_genome.name
+        self.genome = simulated_genome
+        self.sequenced = sequenced_sample
+
+        # TODO: other convenience functions?
+
+def from_list(genomes, sequenced):
+    """ Construct processed samples from lists of SimulatedGenomes and SequencedSamples 
+        Throws an error if the sample names are not consistent.
+    """
+
+    genome_dict = {g.name: g for g in genomes}
+    sequenced_dict = {s.name: s for s in sequenced}
+
+    # Validate
+    if len(genome_dict) != len(genomes):
+        raise Exception("Genomes with non-unique names found")
```

Comment:
I'm probably missing something, but surely len(genome_dict) always equals len(genomes) because of how it's been setup on line 23 ?

Reply:
Not necessarily. If two samples have the same name, one will be lost. Consider this simplified case:
```
>>> dict = {x: x for x in ["a", "b", "b"]}
>>> dict
{'a': 'a', 'b': 'b'}
```

Reply:
Ah! yes nice!